src/main/channel/channel-listeners/start-listening-of-channels.injectable.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import { onLoadOfApplicationInjectionToken } from "../../start-main-application/runnable-tokens/on-load-of-application-injection-token";
+import listeningOfChannelsInjectable from "../../../common/sync-box/channel/listening-of-channels.injectable";
+
+const startListeningOfChannelsInjectable = getInjectable({
+  id: "start-listening-of-channels-main",
+
+  instantiate: (di) => {
+    const listeningOfChannels = di.inject(listeningOfChannelsInjectable);
+
+    return {
+      run: async () => {
+        await listeningOfChannels.start();
```

Comment:
Why is start async? All of the registrations are sync

Reply:
There is no need to limit ourself to synchronous stuff. There was use case (the dynamic icon) which introduced the asynchronicity but since that's gone, we could remove it if we so decide. Not part of this PR anyway.